src/App.js

```diff
@@ -2,15 +2,16 @@ import "./App.css";
 import { Header } from "./component/";
 import { Footer } from "./component/";
 import { Route, Routes } from "react-router-dom";
-import { Home } from "./pages/";
-import { VideoListing } from "./pages/";
+import { Home, VideoListing, Like } from "./pages/";
+
 function App() {
   return (
     <div className="App">
       <Header />
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/videoList" element={<VideoListing />} />

```

Comment:
you can make a seperate component for Route

Reply:
okay